### PR TITLE
Simplify notifications and implement multiple telegram channels

### PR DESCRIPTION
**Pull request by KillerX**
* Removed the `notificationService` concept
* Removed the `notifications.Target` concept -> You have to send emails and telegram messages separately
* Added support for multiple channels
* The util functions for notifications now log errors but don't return them as they were generally ignored

Fixes https://github.com/bcc-code/bcc-media-flows/issues/204

### CODE REVIEW

File: utils/workflows/files.go [hunk 1]
```diff
@@ -191,7 +192,7 @@ func RcloneWaitForFileGone(ctx workflow.Context, file paths.Path, retries int) e
 			break
 		}
 
-		NotifyTelegramChannel(ctx, fmt.Sprintf("⚠️ File ```%s``` already exists, retrying in one minute (%d/%d)", file.Rclone(), i, retries))
+		NotifyTelegramChannel(ctx, telegram.ChatOslofjord, fmt.Sprintf("⚠️ File ```%s``` already exists, retrying in one minute (%d/%d)", file.Rclone(), i, retries))
```
Comment (andreasgangso) [inline utils/workflows/files.go:195]: maybe not

**Review by andreasgangso — COMMENTED** [review]
I think the abstractions that was there was good conceptually, but since we still haven't really had a use for it yet I agree to remove it. We can always add it back later.

**Review by andreasgangso — CHANGES_REQUESTED** [review]
see above files.go

File: utils/workflows/files.go (hunk 1 above)
Comment (KillerX) [reply utils/workflows/files.go:195]: Fixed in See #224

**Pull request merged**